CK2Plus_expanded/interface/000_CK2Plus_expanded_societies.gfx

```diff
@@ -31,4 +31,36 @@ spriteTypes = {
 		name = "GFX_society_currency_warrior_lodge_celtic"
 		textureFile = "gfx\\interface\\societies\\society_currency_warriors.dds"
 	}
+
+	# The Protectors of Mari
+	spriteType = {
+		name = "GFX_society_main_background_warrior_lodge_basque"
+		textureFile = "gfx\\interface\\societies\\society_bg_fianna.dds"
+	}
+
+	spriteType = {
+		name = "GFX_society_expansion_background_warrior_lodge_basque"
+		textureFile = "gfx\\interface\\societies\\society_exp_bg_band_of_medeina.dds"
+	}
+
+	spriteType = {
+		name = "GFX_icon_small_warrior_lodge_basque"
+		textureFile = "gfx\\interface\\societies\\symbol_small_band_of_medeina.dds"
+		noOfFrames = 2
+	}
+
+	spriteType = {
+		name = "GFX_society_portrait_ring_warrior_lodge_basque"
+		textureFile = "gfx\\interface\\societies\\portrait_ring_band_of_medeina.dds"
+	}
+
+	spriteType = {

```

Comment:
oh sorry, inadvertently included this. this'll change once I get some basque gfx from @maalunar.